install/preview/entrypoint.sh

```diff
@@ -54,45 +54,121 @@ cat "${HOME}"/.local/share/mkcert/rootCA.pem >> /etc/ssl/certs/ca-certificates.c
 # also send root cert into a volume
 cat "${HOME}"/.local/share/mkcert/rootCA.pem > /var/gitpod/gitpod-ca.crt
 
-cat << EOF > /var/lib/rancher/k3s/server/manifests/ca-pair.yaml
+FN_CACERT="./ca.pem"
+FN_SSLCERT="./ssl.crt"
+FN_SSLKEY="./ssl.key"
+
+cat "${HOME}"/.local/share/mkcert/rootCA.pem > "$FN_CACERT"
+mkcert -cert-file "$FN_SSLCERT" \
+  -key-file "$FN_SSLKEY" \
+  "*.ws.${DOMAIN}" "*.${DOMAIN}" "${DOMAIN}" "reg.${DOMAIN}" "registry.default.svc.cluster.local" "gitpod.default" "ws-manager.default.svc" "ws-manager" "ws-manager-dev" "registry-facade" "server" "ws-manager-bridge" "ws-proxy" "ws-manager" "ws-daemon.default.svc" "ws-daemon" "wsdaemon"
```

Comment:
Looks like this is enough, and we don't have to generate induvidual certs for each purpose.